lib/util/SortableSet.js

```diff
@@ -3,6 +3,7 @@
 /**
  * A subset of Set that offers sorting functionality
  * @template T item type in set
+ * @extends {Set<T>}
```

Comment:
[Typings Guide -section `Class`](https://github.com/webpack/webpack/issues/7122)
Or is this rule applicable only if you inherit from custom types?

Reply:
I wish I didn't have to have `@extend` but generic type is not inferred

Reply:
Typescript needs to know that SortableSet<T> extends Set<T>, and there's no way to specify the type variables in Javascript. Probably the rule needs an exception.

Reply:
By the way, Microsoft/TypeScript#25101 will be fixed in typescript@next tomorrow.